lnrpc/lightning.proto

```diff
@@ -2116,18 +2116,104 @@ message BatchOpenChannel {
     */
     string close_address = 7;
 
+    /*
+    Funding shims are an optional argument that allow the caller to intercept
+    certain funding functionality. For example, a shim can be provided to use a
+    particular key for the commitment key (ideally cold) rather than use one
+    that is generated by the wallet as normal, or signal that signing will be
+    carried out in an interactive manner (PSBT based).
+    */
```

Comment:
I am unsure if the funding shim is expected to work with the batch channel open flow which is itself interactive. Maybe @guggero can opine here?